swap_meet/decor.py

```diff
@@ -1,2 +1,8 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, condition = 0.0):
+        super().__init__(category = "Decor", condition = condition)
```

Comment:
One small stylistic note: don't include spaces on either side of the equals sign for default arguments. We would instead write:

```
def __init__(self, condition=0.0):
        super().__init__(category="Decor", condition=condition)
```